src/Accordion/Accordion.css

```diff
@@ -0,0 +1,68 @@
+.accordion-wrapper {
+  justify-content: center;
+  width: 100%;
+}
+
+.accordion-item {
+  border: 2px solid #333;
+  border-radius: 4px;
+  margin: 5px 0;
+  padding: 2px 4px;
+}
+
+.accordion-title {
+  display: flex;
+  background-size: 100%;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 14px;
+  font-family: "Roboto Slab", serif;
+}
+
+.accordion-plus-sign{
+  content: "";
+  background-color: black;
+  width: 1.5px;
```

Comment:
same here, all of width and position definitions comes before colors/margin/paddings but that for the future to have in mind